e2e/rbd.go

```diff
@@ -217,6 +221,22 @@ var _ = Describe("RBD", func() {
 				}
 			}
 			deployRBDPlugin()
+			deployment, err := f.ClientSet.AppsV1().Deployments(cephCSINamespace).Get(
```

Comment:
would be good to place this in a helper function, and do it in a `wait.PollImmediately()` function, including retries for certain failures (see `e2e/util.go` for examples).